otter/test/rest/request.py

```diff
@@ -313,6 +313,25 @@ def test_invalid_methods_are_405(self):
             self.assert_status_code(405, method=method)
 
 
+def setup_mod_and_trigger(testcase):
```

Comment:
The main reason was that not all tests inheriting from mixin need to mock the controller. Ideally, it shouldn't affect even if it is mocked but I thought of keeping it separate for this reason.